dockerfiles/alpine/obam/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # ------------------------------------------------------------------------
 #
-# Copyright 2019 WSO2, Inc. (http://wso2.com)
+# Copyright 2020 WSO2, Inc. (http://wso2.com)
```

Comment:
@thiloshon let's not change the license header year, because we had the copy of this file even in a previous release.

This is applicable to all files in this PR.

Reply:
Fixed!